### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: ProtonVPNTests/ViewModels/Signup/SignUpFormViewModelTests.swift [hunk 1]
```diff
@@ -111,7 +111,7 @@ class SignUpFormViewModelTests: XCTestCase {
         viewModel.password2 = "abc"
         
         userApiService.callbackcheckAvailability = { username, success, failure in
-            failure(ApiError.uknownError)
+            failure(ApiError.unknownError)
```
Comment (jsoref) [inline ProtonVPNTests/ViewModels/Signup/SignUpFormViewModelTests.swift:114]: This presumably will break as it appears to be dependent on some external thing. I'm not sure where it is...

File: ProtonVPN/ViewModels/Human Verification/VerificationEmailViewModel.swift [hunk 2]
```diff
@@ -75,7 +75,7 @@ class VerificationEmailViewModel {
             }
                     
             switch (apiError.httpStatusCode, apiError.code) {
-            case (_, ApiErrorCode.signupWithProtonMailAdress):
+            case (_, ApiErrorCode.signupWithProtonMailAddress):
```
Comment (jsoref) [inline ProtonVPN/ViewModels/Human Verification/VerificationEmailViewModel.swift]: This presumably will break as it appears to be based on some external thing

**Review by jsoref — COMMENTED** [review]
I try to avoid changing third party modules. If any of these files are, please let me know.

**Comment by jaroslavas** [comment]
Thank You for contribution. Your fixes will be merged on the next release.

**Pull request closed without merging**